active-tasks/react-class-components.json

```diff
@@ -47,11 +47,6 @@
       "text": "TypeScript isn't used",
       "max": -95
     },
-    {
-      "type": "penalty",
-      "text": "eslint-plugin-react-compiler isn't used or isn't configured to throw errors",
-      "max": -50
-    },
```

Comment:
I am disagree with it. Because we continue to work with previous task and eslint should work correctly.